tests/rule_test.py

```diff
@@ -121,7 +120,7 @@ def test_simple(self, simple_source):
   write( output_unit, '("Hello ", A)' ) 'world'
   
 end program some_trailing_whitespace
-'''
+'''  # noqa: W291, W293
```

Comment:
What is it about this string which requires it to be exempted from QA?

Reply:
This is to allow flake8 to ignore trailing white spaces in its own line or at the end of a statement.

Reply:
In the tests for trailing white space, I understand.

Reply:
Yes. The multi-line strings has trailing white spaces in a line and white spaces in one of the empty lines, so need both of these directives to ignore.